fmriprep/workflows/bold/base.py

```diff
@@ -462,6 +473,87 @@ def init_bold_wf(
             (bold_anat_wf, bold_surf_wf, [("outputnode.bold_file", "inputnode.bold_t1w")]),
         ])  # fmt:skip
 
+    if config.workflow.cifti_output:
+        from .resampling import init_bold_fsLR_resampling_wf, init_bold_grayords_wf
+
+        bold_MNI6_wf = init_bold_volumetric_resample_wf(
+            metadata=all_metadata[0],
+            fieldmap_id=fieldmap_id if not multiecho else None,
+            omp_nthreads=omp_nthreads,
+            name='bold_MNI6_wf',
+        )
+
+        bold_fsLR_resampling_wf = init_bold_fsLR_resampling_wf(
+            estimate_goodvoxels=config.workflow.project_goodvoxels,
+            grayord_density=config.workflow.cifti_output,
+            omp_nthreads=omp_nthreads,
+            mem_gb=mem_gb["resampled"],
+        )
+
+        bold_grayords_wf = init_bold_grayords_wf(
+            grayord_density=config.workflow.cifti_output,
+            mem_gb=mem_gb["resampled"],
+            repetition_time=all_metadata[0]["RepetitionTime"],
+        )
+
+        ds_bold_cifti = pe.Node(
+            DerivativesDataSink(
+                base_directory=fmriprep_dir,
+                space='fsLR',
+                density=config.workflow.cifti_output,
+                suffix='bold',
+                compress=False,
+                TaskName=all_metadata[0].get('TaskName'),
+                **prepare_timing_parameters(all_metadata[0]),
```

Comment:
not necessarily related to this PR, but is this the only time we are outputing this metadata? seems like something we should only calculate once and then distribute to any bold outputs

Reply:
No, we also calculate it in the volumetric workflow:

https://github.com/nipreps/fmriprep/blob/c2940289829c2aec96c144f3a4938744c33a04d5/fmriprep/workflows/bold/outputs.py#L686-L737

So it gets recalculated several times. We could refactor the function to have a cacheable core (can't `@lru_cache` with a dictionary argument), but I'm not sure saving <1s (estimate, not measured) at workflow build is really worth it.